src/main/resources/application.properties

```diff
@@ -4,4 +4,12 @@ db.janitor.uri=jdbc:postgresql://127.0.0.1:5432/${DATABASE_NAME}
 db.janitor.username=${DATABASE_USER}
 db.janitor.password=${DATABASE_USER_PASSWORD}
 db.janitor.recreateDbOnStart=true
```

Comment:
unrelated question: I had `recreateDbOnStart = true` Because I was activaitly work on db things(also in GKE), and created bunch of dirty record, so I set this to true for now. Do you think we should keep as it or change it back to false?

Reply:
I think we'll need to change it back by the time we have persistent environments, but I don't think that's just yet.

Reply:
then do we need to do the similar things for the staritway db?

Reply:
Yes, eventually we need to change the values as well.
Note that Stairway#initialize does the recreate/migration, so we are calling that code already in this PR. https://github.com/DataBiosphere/stairway/blob/925d0724f57bc881e9f7bd1588bebd3818cc1732/src/main/java/bio/terra/stairway/Stairway.java#L351-L359